### DISCUSSION TITLE
Additional features: bpm and multiple genres per track

### DISCUSSION THREAD

**Discussion by freugo54** [strawberrymusicplayer/strawberry#979] [Ideas]
I am still using Clementine although it seems Strawberry has resolved a couple of issues that exist with Clementine. One of the reasons why I don't change to Strawberry is that Clementine offers a field for for bpm / beats per minute. This allows sorting playlists by speed, very helpful e.g. for dance parties or training. The field is not editable in Clementine, so I use a DJ software (Mixxx) to fill it, however for the event itself I find that software less convenient than Clementine. 
Would you mind adding this feature to Strawberry as well?
Another idea for a new feature would be the possibility to assign multiple genres to one single track, allowing it to appear in different places of the library and to search for them independently. Sometimes compositions are not clearly attributable to one specific genre or you can dance different styles to the them, so to identify suitable pieces of music such feature would be very useful. Currently I help myself by putting eligible keywords altogether in one genre entry but this is a pretty dumb and dirty workaround creating new genres for each combination, possibly even more when changing the sequence erroneously. Very unprofessional I must admit. Can this be a new feature for Strawberry? I would love it!

**Comment by jonaski** [comment]
Add this to https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
